ikenna-portfolio/portfolio/src/main/webapp/script.js

```diff
@@ -12,160 +12,76 @@
+  document.getElementById("projects-pics").innerHTML = project.imageHTML;
+  document.getElementById("pro-desc").innerHTML = project.desc;
+  document.getElementById("pro-links").innerHTML = project.links;
 }
 
-/**
-* Changes Calisthenics Container to Back Lever state
-*/
-function changeBackLever() {
-  document.getElementById("cal-image1").src = "/images/back_lever.jpg";
-  document.getElementById("cal-image2").src = "/images/adv_tuck_back_lever.jpg";
-  document.getElementById("goal").innerText = goal.replace("-", "Back Lever");
-  document.getElementById("current").innerText = current.replace("-", "Advanced Tuck Back Lever");
+async function loadCalisthenicsContainer() {
+  const response = await fetch("/calisthenics");
+  const calisthenics = await response.json();
+  document.getElementById("cal-image1").src = calisthenics.img1;
+  document.getElementById("cal-image2").src = calisthenics.img2;
+  document.getElementById("goal").innerText = goal.replace("-", calisthenics.title1);
+  document.getElementById("current").innerText = current.replace("-", calisthenics.title2);
 }
 
-/**
-* Changes Calisthenics Container to Pistol Squat state
-*/
-function changePistolSquat() {
-  document.getElementById("cal-image1").src = "/images/pistol_squat.jpg";
-  document.getElementById("cal-image2").src = "/images/supported_pistol_squat.jpg";
-  document.getElementById("goal").innerText = goal.replace("-", "Pistol Squat");
-  document.getElementById("current").innerText = current.replace("-", "Pole Supported Pistol Squat");
+async function loadCommentsContainer() {
+  const dresponse = await fetch("/data");
+  const comments = await dresponse.json();
+  const mresponse = await fetch("/count")
+  const metadata = await mresponse.json(); 
+  let msg = "";
+  document.getElementById('comment-count').innerHTML = comment_count.replace("-", metadata.count);
+  document.getElementById('filter-label').innerHTML = current_filter.replace("-", getFilter(metadata.ascending, metadata.search));
+  document.getElementById('page-count').innerHTML = page_count.replace("page_num", (metadata.page + 1)).replace("page_max", metadata.max_pages); 
+  for(comment of comments) {
+    if(comment.name === "") continue;
+    msg += comment_item.replace("timestamp", new Date(comment.timestamp)).replace("comment_id", comment.id).replace("comment_name", comment.name).replace("comment_text", comment.text);
+  }
+  document.getElementById("comments").innerHTML = msg;
 }
 
-/**
-* Changes Calisthenics Container to Front Lever state
-*/
-function changeFrontLever() {
-  document.getElementById("cal-image1").src = "/images/front_lever.jpg";
-  document.getElementById("cal-image2").src = "/images/single_leg_front_lever.png";
-  document.getElementById("goal").innerText = goal.replace("-", "Front Lever");
-  document.getElementById("current").innerText = current.replace("-", "Single Leg Front Lever");
+function getFilter(ascending, search) {
```

Comment:
How about moving this code to MetaData class so that all related logic are in one place.

Reply:
done